harness/reporters/reporter.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2017 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package reporters
+
+import (
+	"time"
+)
+
+const (
+	TestResultFail TestResult = "FAIL"
```

Comment:
based on this also being used in the `harness` pkg, it seems plausible this should be in `harness/testresult` and have the consts renamed to `Fail`/`Skip`/`Pass`.

As-is, it's not too bad. I'm okay with it just living here too.

Reply:
I don't have a particular preference on this, so pushing up the changes. I initially left this out because it felt a bit wonky having a package that only had a struct definition and three constants.

Good to merge on my end.